cli/tests/error_worker_permissions_remote.ts

```diff
@@ -0,0 +1,5 @@
+const w = new Worker(
+  new URL("http://localhost:4545/cli/tests/subdir/worker_types.ts").toString(),
```

Comment:
The `new URL().toString()` isn't really needed here.

Reply:
🤦